container.go

```diff
@@ -160,12 +174,6 @@ func (c *container) Image(ctx context.Context) (Image, error) {
 	}, nil
 }
 
-func (c *container) Extensions() map[string]prototypes.Any {
-	c.mu.Lock()
-	defer c.mu.Unlock()
-	return c.c.Extensions
-}
-
 func (c *container) NewTask(ctx context.Context, ioCreate IOCreation, opts ...NewTaskOpts) (Task, error) {
```

Comment:
RLock seems good enough.

Reply:
i don't want this to be called twice since you can only have 1 task per container